recipes/python-ironicclient/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "python-ironicclient" %}
+{% set version = "1.16.0" %}
+{% set sha256 = "cab1dbc0d10d2a5b82e43f3cc2a7cc9d85e5b79eb2416ef8247ae7162c98dbf3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python
+    - pbr
+
+  run:
+    - python
+    - pbr
+    - keystoneauth1
+    - oslo.utils
+    - oslo.i18n
+    - oslo_serialization
+    - prettytable
+    - six
+
+test:
+  imports:
+    - ironicclient
+  commands:
+    - ironic help
+
+about:
+  home: https://docs.openstack.org/python-ironicclient/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Python Bindings to the OpenStack Ironic API
+  dev_url: https://github.com/openstack/python-ironicclient
```

Comment:
According to PyPI the source is at ( https://git.openstack.org/cgit/openstack/python-ironicclient ).